### ISSUE TITLE
Unable to backward when training

### ISSUE THREAD

**Issue by VietPT3502** [zcemycl/PyTorch-DeepFloorplan#4]
Hello, i have try to train using your code. And when loss.backward(), this take forever to run a single iter. When i tried to use another basic model, for example Unet the problems is fixed.
So is there any problems happening in your model's code and can you give me some advice.
Thanks for advance!

**Comment by LTayfaker** [comment]
When I trained I found that the training of the room type branch didn't work, the room predict mask was abnormal, but the training of the room boundaries was normal.

**Comment by zcemycl** [comment]
Tune your batch size or learning rate, sometimes if the training on boundary appears to be way faster than the training on room type. Room type would appear black.
